flake.nix

```diff
@@ -21,6 +21,8 @@
     nix-index-database.url = "github:nix-community/nix-index-database/main";
     nixos-hardware.url = "github:NixOS/nixos-hardware/master";
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    pop-os-default-settings.url = "github:pop-os/default-settings/master_noble";
```

Comment:
default branch is `master_jammy`, which is older, and doesn't have that `fs.inotify.max_user_instances` change